src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,16 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer {
+    @Override
+    public List<Machine> get() {
```

Comment:
```suggestion
    public List<Bulldozer> get() {
```
кожен продюсер має повертати свій ліст